backend/routes/users.js

```diff
@@ -63,10 +77,14 @@ router.post('/login', (req, res) => {
             // This is how to connect to Blockchain network with userID
             let connection = await network.connectToNetwork(user._id.toString());
 
-            res
-                .header('x-refresh-token', authTokens.refreshToken)
-                .header('x-access-token', authTokens.accessToken)
-                .send(user);
+            if (user.confirmed === true) {
+                res
+                    .header('x-refresh-token', authTokens.refreshToken)
+                    .header('x-access-token', authTokens.accessToken)
+                    .send(user);
+            } else {
+                res.status(400).send("User not confirmed.");
```

Comment:
Bit confusing. How about "Email not confirmed yet, Please check your registered email" ?

Reply:
I don't think this is being shown in the ui so it shouldn't matter too much. A more detailed message is being shown to users https://github.com/ECE493W2021T3/eVoter/pull/73/files#diff-b0d012490e1f819489a45bc9aab9ca27cb0b094d1cc3700aaad07df2ce7dc328R71-R72